node/api/api_encoding.md

```diff
@@ -32,6 +32,7 @@ tag.
 | Prefix  | Encoding | Description |
 | ---     | ---      | --- |
 | ak      | base58   | Account pubkey |
+| ba      | base64   | Byte array |
```

Comment:
This brought in by https://github.com/aeternity/aeternity/pull/2415/files#diff-31d7a50c99c265ca2793c20961b60979R68
Ok.